test/types/void/void-variable-use-3.good

```diff
@@ -1 +1 @@
-void-variable-use-3.chpl:5: error: cannot use the value of a void variable
+
```

Comment:
Right, it prints nothing as implemented.  I took the approach that a void variable or field is something you wanted to be removed from the code, and something that is not there shouldn't cause anything to print.  I agree that it could also be nice to go the other way and print 'void'.  It's not obvious to me which is better.